.github/workflows/hotfix-wf-service.yml

```
@@ -0,0 +1,400 @@
+        uses: aquasecurity/trivy-action@master
+        continue-on-error: true
+        with:
+          cache-dir:
+          image-ref: ${{ steps.docker-version.outputs.full_image }}
+          format: 'table'
+          ignore-unfixed: true
+          exit-code: 1
+          trivyignores: ./.trivyignore
+          vuln-type: 'os,library'
+          severity: 'CRITICAL'
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)